config/locales/devise.en.yml

```diff
@@ -1,63 +0,0 @@
-# if me: DO NOT modify this file. if you need to add translations, add them to en.yml
-# Additional translations at https://github.com/plataformatec/devise/wiki/I18n
-
```

Comment:
@vinzruzell any reason why all these were deleted?

Reply:
what u mean sir?

Reply:
Ahh okay I thats the problem. I have just copied it from my first commit.